idea/testData/inspections/forEachParameterNotUsed/inspectionData/inspections.test

```diff
@@ -0,0 +1 @@
+// INSPECTION_CLASS: org.jetbrains.kotlin.idea.inspections.ForEachParameterNotUsed
```

Comment:
Here you have to add `// WITH_RUNTIME` line, otherwise all your tests fail. In the future, please re-run your own tests before re-submitting PR.

Reply:
(comment: this line adds full Kotlin SDK to test project, without it `forEach` function is not visible).